mittab/libs/data_import/import_teams.py

```diff
@@ -46,18 +48,35 @@ def import_teams(fileToImport):
             team_errors.append('ERROR: Insufficient Columns in Sheet. No Data Read')
             return team_errors
 
+    # verify no duplicate debaters, give error messages
+    deb_indicies = []
     for i in range(1, num_teams):
+        deb_indicies.append((sh.cell(i, 3).value.strip(), i))  # tuple saves debater name and row
+        deb_indicies.append((sh.cell(i, 7).value.strip(), i))
+    deb_names = [i[0] for i in deb_indicies]
+    names_dict = collections.Counter(deb_names)
+    for deb_index in deb_indicies:
+        if names_dict.get(deb_index[0]) > 1:  # if dict says appears more than once
+            # inform that duplicate exists at location, report relevant information
+            row_num = deb_index[1]
+            msg = "Check for duplicate debater " + deb_index[0] + " in team " + sh.cell(row_num, 0).value + \
+                  ", on XLS file row " + str(row_num)
+            team_errors.append(msg)
 
+    for i in range(1, num_teams):
+
+        # Name, School, Seed [full, half, free, none], D1 name, D1 v/n?, D1 phone, D1 prov,
+        # D2 name, D2 v/n?, D2 phone, D2 prov
+
+        # team name, check for duplicates
+        duplicate = False
         team_name = sh.cell(i, 0).value
         if team_name == '':
-            team_errors.append('Row ' + str(i) + ': Empty Team Name')
+            team_errors.append('Skipped row ' + str(i) + ': empty Team Name')
             continue
-        try:
-            Team.objects.get(name=team_name)
-            team_errors.append(team_name + ': Duplicate Team Name')
-            continue
-        except:
-            pass
+        if Team.objects.filter(name=team_name).first() is not None:  # inform that duplicates exist
```

Comment:
Same thing as the room import: The Django model already does this for us

Reply:
I'm not sure exactly to what this refers now. Could you clarify?